### PR TITLE
Integrate helloworld project with Travis CI

### PR DESCRIPTION
**Pull request by rajdeepd**
**Related issues**
https://github.com/salesforce/TransmogrifAI/issues/114

**Describe the proposed solution**
Added helloworld samples to the travis build

**Describe alternatives you've considered**
none

**Additional context**
none

### CODE REVIEW

File: helloworld/build.gradle [hunk 1]
```diff
@@ -53,6 +55,8 @@ configurations {
 dependencies {
     // Scala
     zinc 'com.typesafe.zinc:zinc:0.3.15'
+    scoverage "org.scoverage:scalac-scoverage-plugin_$scalaVersion:$scoveragePluginVersion"
```
Comment (tovbinm) [inline helloworld/build.gradle]: you also need 
```
testCompile "org.scoverage:scalac-scoverage-plugin_$scalaVersion:$scoveragePluginVersion"
testCompile "org.scoverage:scalac-scoverage-runtime_$scalaVersion:$scoveragePluginVersion"
```

File: .travis.yml [hunk 2]
```diff
@@ -39,6 +39,20 @@ jobs:
         sed -i 's/\\n$//g' "$TESTS_FILES_PREFIX"*
         popd
         ./gradlew scalaStyle testScoverageClasses --parallel
+    - stage: 'compile helloworld project'
+      name: 'compile helloworld project'
+      script: |
+        find **/helloworld/src/test/scala/* -type f -name '*.scala' | cut -d "/" -f5-$2 | sed 's/.\{6\}$/\*/g' > $TESTS_FILE_NAME
```
Comment (tovbinm) [inline .travis.yml]: I dont think any of that complexity for helloworld. Simply do `cd helloworld && ./gradlew scalaStyle testScoverageClasses --parallel"

Comment (rajdeepd) [reply .travis.yml]: fixed

File: helloworld/build.gradle (hunk 1 above)
Comment (rajdeepd) [reply helloworld/build.gradle]: fixed

File: .travis.yml [hunk 3]
```diff
@@ -68,3 +72,10 @@ jobs:
         echo -e "Test files running on this container : `eval $COUNT_TEST_FILE_CMD`\n$TEST_FILES\n"
         ./gradlew reportScoverage
       name: 'Build Part 4'
+    - script: |
```
Comment (tovbinm) [inline .travis.yml]: simply do:
```
- script: |
      cd helloworld && ./gradlew reportScoverage
name: 'Build - Helloworld'

```

Comment (rajdeepd) [reply .travis.yml]: @tovbinm fixed

**Pull request merged**